JS/Form-validation.js

```diff
@@ -0,0 +1,20 @@
+const formelement = document.getElementsByTagName('form');// eslint-disable-line
```

Comment:
Please ensure not to disable the linter as they are important to enforce best practices and to catch syntax errors when writing your code. Kindly put it back on, resolve the issue, and submit it for review.

Reply:
- [ ] Please make the change as requested by the previous code reviewer. Do not disable any linter checks as it is not a good practice.